test/server/models/Update.test.js

```diff
@@ -58,4 +59,14 @@ describe('server/models/Update', () => {
       expect(update.dataValues.isPrivate).to.equal(true);
     });
   });
+
+  describe('delete update', () => {
+    it('frees up current slug when deleted', async () => {
+      const slug = 'unique-slug';
+      const update = await models.Update.findByPk(5);
+      await update.destroy();
+
+      expect(slug).to.not.be.equal(update.slug);
```

Comment:
If I add a `console.log(update.slug);` after this line, I get `unique-slug1` as a result, which is not what we want. Can we improve the test to make sure the output slug is formatted as we expect? If you need to reload soft-deleted data at some point, you can pass `paranoid: false` to sequelize

Reply:
fixed and improved test too.

![Screenshot at 2021-01-22 16-54-20](https://user-images.githubusercontent.com/46647141/105485632-39a68000-5cd3-11eb-80a0-2aedfe5de200.png)